### ISSUE TITLE
About UMATapplication

### ISSUE THREAD

**Issue by Gaoliu19910601** [KratosMultiphysics/Kratos#5686]
Hi everyone
   Does anyone know how to  use UMAT in applications such PFEM or MPM ? I cannot find any example about this. Or is it just in development and not ready for use?

**Comment by philbucher** [comment]
@lluis-mv @KratosMultiphysics/mpm @KratosMultiphysics/pfem @KratosMultiphysics/pfem2

**Comment by lluis-mv** [comment]
Dear @Gaoliu19910601 ,

The UmatApplication relies on the ConstitutiveModelApplication. Basically, all the constitutive models in the application have as base class the ConstitutiveLaw of Kratos, so, I guess it could be used in any application, though I have only tested in SolidMechanicsApplication and PfemSolidMechanicsApplication.

Recently I left UPC and I moved abroad and changed my research topic (I am no longer using Kratos in a daily basis) so I could not assist you a lot. There is a small test in the application, and adding a new Umat model does not require lots of coding, everything is very encapsulated.
